man/systemd.link.xml

```diff
@@ -140,28 +144,32 @@
           is set (or if prefixed with the exclamation mark unset). See
           <varname>ConditionKernelCommandLine=</varname> in
           <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-          for details.</para>
-        </listitem>
-      </varlistentry>
-        <varlistentry>
-          <term><varname>KernelVersion=</varname></term>
-          <listitem>
-            <para>Checks whether the kernel version (as reported by <command>uname -r</command>) matches a certain
-            expression (or if prefixed with the exclamation mark does not match it). See
-            <varname>ConditionKernelVersion=</varname> in
-            <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry> for
-            details.
-            </para>
-          </listitem>
-        </varlistentry>
+          for details. When prefixed with an exclamation mark (<literal>!</literal>), the result is nagated.
+          When specified more than once, the results are ANDed. If an empty string is assigned,
+          then previously assigned values are cleared.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>KernelVersion=</varname></term>
+        <listitem>
+          <para>Checks whether the kernel version (as reported by <command>uname -r</command>) matches a certain
+          expression (or if prefixed with the exclamation mark does not match it). See
```

Comment:
This information about negation is now repeated needlessly. I think the part in parentheses should be removed.